tokio-macros/src/lib.rs

```diff
@@ -125,7 +141,36 @@ use proc_macro::TokenStream;
 /// }
 /// ```
 ///
-/// ### Set number of worker threads
+/// ## Using the local runtime
+///
+/// Available in the [unstable API][unstable] only.
+///
+/// The [local runtime] is similar to the current-thread runtime but
+/// supports [`task::spawn_local`](../tokio/task/fn.spawn_local.html).
+///
+/// ```ignore
+/// #[tokio::main(flavor = "local")]
+/// async fn main() {
+///     println!("Hello world");
+/// }
+/// ```
+///
+/// Equivalent code not using `#[tokio::main]`
+///
+/// ```ignore
```

Comment:
Does this work?
```suggestion
/// ```
/// #![cfg(tokio_unstable)]
```

Reply:
If we can execute it, then I think we should. Why can't we?

Reply:
We can do 352fa30fa2764c0fa90fbbcc60e089bd800de75c.

Reply:
Right, that's https://github.com/tokio-rs/tokio/commit/b040de5d053649147e2210fa3d0f9688ffe7fbb8 :)